packages/big-design/src/components/Alert/spec.tsx

```diff
@@ -35,26 +36,24 @@ test('render info Alert', () => {
 });
 
 test('renders with link', () => {
-  const { queryByRole, container } = render(
-    <Alert messages={[{ text: 'Success', link: { text: 'Link', href: '#' } }]} />,
-  );
+  const { container } = render(<Alert messages={[{ text: 'Success', link: { text: 'Link', href: '#' } }]} />);
 
   expect(container.firstChild).toMatchSnapshot();
 
-  const link = queryByRole('link') as HTMLAnchorElement;
+  const link = screen.getByRole('link') as HTMLAnchorElement;
```

Comment:
Let's remove these casts and use `await screen.findBy*`. You'll have to convert the test callback to be `async`.
```suggestion
  const link = await screen.findByRole<HTMLAnchorElement>('link');
```